options.go

```diff
@@ -57,8 +57,8 @@ type Options struct {
 	// Maximum number of retries before giving up.
 	// Default is to not retry failed queries.
 	MaxRetries int
-	// Whether to retry queries cancelled because of statement_timeout.
-	RetryStatementTimeout bool
+	// Whether to retry queries cancelled because of statement_timeout or cancel request.
+	RetryCancelledQuery bool
```

Comment:
Renaming this would be a breaking change.  I recommend we leave the name the same.

Reply:
Good point! I will revert the change to keep as it is.